cmd/gb/gb_test.go

```
@@ -451,7 +451,7 @@ func (t *T) mustBeEmpty(root string) {
 		found = append(found, path)
 		return nil
 	}
-	err = filepath.Walk(root, fn)
+	_ = filepath.Walk(root, fn)
```

this is a bug, err is not being checked.